task/source-build/0.1/source-build.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: source-build
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "appstudio, hacbs"
+spec:
+  description: Source image build.
+  params:
+    - name: BINARY_IMAGE
+      description: The corresponding binary image based on which to generate source image.
+      type: string
+    - name: BASE_IMAGES
+      description: >-
+        Base images used to build the binary image. Each image per line in the same order of FROM
+        instructions specified in a multistage Dockerfile. Default to an empty string, which means
+        to skip handling a base image.
+      type: string
+      default: ""
+  results:
+    - name: BUILD_RESULT
+      description: Build result.
+    - name: SOURCE_IMAGE_URL
+      description: The source image url.
+    - name: SOURCE_IMAGE_DIGEST
+      description: The source image digest.
+  workspaces:
+  - name: workspace
+    description: Workspace where task works inside. Source code is included in this workspace and any artifacts generated by this task are also stored here.
+  volumes:
+    - name: source-build-work-place
+      emptyDir:
+        medium: Memory
```

Comment:
Is the description for `workspace` correct? Doesn't the task work inside the source-build-work-place volume, which is a different volume than the workspace?

Reply:
Updated.